mods/ra/maps/exodus/rules.yaml

```diff
@@ -0,0 +1,185 @@
+Player:
+	MissionObjectives:
+	PlayerResources:
+		DefaultCash: 5000
```

Comment:
Err, slight correction. Because of [https://github.com/OpenRA/OpenRA/pull/11679/files#diff-c170037b524787407d1f07a0151d1d71R312](this check), you actually do want to set the `Cooperative` flag on `MissionObjectives`.

Background: You can leave the flag unset if you're sure that you will always change both players objectives the same way at the same time, but in the linked case, only one player's objective would be changed.